app/components/addOutlet/addOutletController.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+angular.module('batApp')
+    .controller('addOutletController', function (getDataFactory, $state) {
+      var vm = this
+      vm.radioData = [
+          { label: 'gold23', value:1 },
```

Comment:
sir i have created the json of radio button and rendering on html page using ng-repeat